### PR TITLE
ASI library install script

### PR DESCRIPTION
**Pull request by danjampro**
Add a convenience script for installing ASI libraries.

### CODE REVIEW

File: scripts/install_asi_libs.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+set -eu
+
+export CAM_LIB_VERSION=1.15.0617
+export EFW_LIB_VERSION=1.5.0615
+
+export BUILD_DIR=$HOME/ASIBuild
```
Comment (lspitler) [inline scripts/install_asi_libs.sh]: Building something off the home directory kind of pollutes it. Suggest using `/tmp/`:
```suggestion
export BUILD_DIR=/tmp/ASIBuild
```

**Review by lspitler — APPROVED** [review]
one suggestion

File: scripts/install_asi_libs.sh (hunk 1 above)
Comment (danjampro) [reply scripts/install_asi_libs.sh]: @lspitler  Good point, although I'm not sure that we can always rely on `/tmp` existing in the docker images (I had that problem before). We probably want to get rid of the build dir anyway to save space on the pis, so instead we can just delete it?

File: scripts/install_asi_libs.sh [hunk 2]
```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
```
Comment (wtgee) [inline scripts/install_asi_libs.sh]: I had sort of redone this already:

```sh
#!/usr/bin/env bash

set -e
set -u

BUILD_DIR=${1:-./ASIBuild}

# Current lib versions.
CAM_LIB_VERSION=1.15.0617
EFW_LIB_VERSION=1.5.0615
# Get the arch -> x86_64 == x86
ARCH=${ARCH:-$(uname -m | cut -d'_' -f1)}

# ZWO camera
function install_zwo() {
  # Install dependencies.
  apt-get update && apt-get --yes install libusb-1.0-0-dev libudev-dev

  mkdir -p "${BUILD_DIR}/zwo" && cd "${BUILD_DIR}/zwo"
  INSTALL_FILE=ASI_linux_mac_SDK_V${CAM_LIB_VERSION}.tar.bz2
  wget "https://astronomy-imaging-camera.com/software/${INSTALL_FILE}"
  tar xvjf "${INSTALL_FILE}" && cd lib
  # Move the library file.
  cp "${ARCH}/libASICamera2.so" /usr/local/lib/
  chmod a+rx /usr/local/lib/libASICamera2.so
  install asi.rules /etc/udev/rules.d

  # ZWO filterwheel
  mkdir -p "${BUILD_DIR}/zwo-filterwheel" && cd "${BUILD_DIR}/zwo-filterwheel"
  INSTALL_FILE=EFW_linux_mac_SDK_V${EFW_LIB_VERSION}.tar.bz2
  wget "https://astronomy-imaging-camera.com/software/${INSTALL_FILE}"
  tar xvjf "${INSTALL_FILE}" && cd lib
  # Move the library file.
  cp "${ARCH}/libEFWFilter.so" /usr/local/lib/
  chmod a+rx /usr/local/lib/libEFWFilter.so
  install efw.rules /etc/udev/rules.d
}

# Make the build dir.
mkdir -p "${BUILD_DIR}"

# Call the function.
install_zwo

# Clean up
rm -rf ${BUILD_DIR}
ldconfig

```

**Review by wtgee — APPROVED** [review]
I had sort of redone this as part of #196, but feel free to merge this.

File: scripts/install_asi_libs.sh (hunk 2 above)
Comment (danjampro) [reply scripts/install_asi_libs.sh]: Looks good! I'll merge your version.

**Pull request merged**